app/views/shared/access_denied.html.erb

```diff
@@ -5,5 +5,6 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl"><%= t('.heading_1') %></h1>
     <p class="govuk-body"><%= t('.permission_text') %></p>
+    <p class="govuk-body"><a href="/providers/applications"><%= t('.back_to_apps_link') %></a></p>
```

Comment:
Can you please use a `link_to` and the correct route helper:
```irb
<p class="govuk-body">
  <%= link_to t('.back_to_apps_link'), providers_legal_aid_applications_path %>
</p>
```
Hard coding a route relative to root can cause problems in the long term - specifically if the app is hosted within a sub-domain.